modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/leases/LeaseTracker.java

```diff
@@ -65,12 +84,22 @@ public class LeaseTracker {
      *
      * @param vaultManager Vault manager.
      * @param msManager Meta storage manager.
+     * @param longLeaseInterval Long lease interval in {@code TimeUnit.MILLISECONDS}.  The interval is used between lease granting attempts.
```

Comment:
It's not obvious from this description that this is the interval for newly granted lease that is not accepted yet, and is never used for actual leases.

Reply:
Because same value is used in LeaseUpdater for the purposes you've mentioned. Here it's used in order to specify max time to await for lease to be populated locally within lease across nodes distribution process. So, to be honest I believe that comment is fine.

Reply:
Same as above, that wasn't part of my changes, I've just used comment from original code. However I don't have any objections to your proposal. Will use the suggestion. Fixed.

Reply:
I do not have any objection. I think the both definition are clear.

Reply:
Fixed,